hack/tools/install_or_tools.sh

```diff
@@ -6,6 +6,8 @@
 cd "${0%/*}"
 source ./common.sh
 
+BUILD_FILE=lib/fzn-or-tools-build
+VERSION_FILE=lib/fzn-or-tools-version
```

Comment:
Suggest to change to ./solver-build and ./solver-version

Reply:
I prefer to leave the tool related name (fzn-or-tools), so it will be easy to connect the installation script and the created file.